examples/produce_assembly_assy/logo.md

```
@@ -0,0 +1,26 @@
+# logo
+
+PartCAD logo
+
+Package: `//pub/examples/partcad/produce_assembly_assy`
+
+<img src="././logo.svg" style="width: auto; height: auto; max-width: 200px; max-height: 200px;">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)